II - Data processing/slides/pandas-data-processing-CE.tex

```diff
@@ -0,0 +1,1181 @@
+		\item Use the following command to import Pandas to your notebook:
+	\end{itemize}
+
+	\hspace{7mm} \texttt{\textcolor{purple}{import} pandas as \textcolor{purple}{pd}}
+
+\end{frame}
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Slide
+\begin{frame}{Pandas}
+
+	\textbf{Dataframes}
+
+	\begin{multicols}{2}
+	
+		\begin{itemize}
+			\item A dataframe is a two-dimensional data structure
+			\item They are very often used to work with data structures where every row represents a unit of observation and every column represents an observation's attribute
+			\item The Stata equivalent to dataframes are datasets
+		\end{itemize}
+		\begin{figure}
+			\centering
+			\includegraphics[width=0.8\linewidth]{img/dataframe.png}
+		\end{figure}
+
+	\end{multicols}
+
+\end{frame}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Slide
+\begin{frame}{Pandas}
+
+	\textbf{Creating a dataframe}
+
+	There are several ways to create a dataframe from scratch. One of the easiest is:
+
+	\begin{enumerate}
+		\item Define a list of strings with the column names
+		\begin{figure}
+			\includegraphics[width=0.8\linewidth]{img/column_names.png}
+		\end{figure}
+		\item Define a list for \textbf{each observation}
+		\begin{figure}
+			\includegraphics[width=0.8\linewidth]{img/observation_lists.png}
+		\end{figure}
+		\item Wrap all of the observation lists in another list
+		\begin{figure}
+			\includegraphics[width=0.8\linewidth]{img/data_list.png}
+		\end{figure}
+
+	\end{enumerate}
+
+\end{frame}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Slide
+\begin{frame}{Pandas}
+
+	\textbf{Creating a dataframe}
+
+	\begin{enumerate}
+		\setcounter{enumi}{3}
+		\item Use the lists \texttt{data} and \texttt{column\_names} as inputs in the \texttt{DataFrame()} command
+		\begin{figure}
+			\includegraphics[width=0.8\linewidth]{img/dataframe_definition.png}
+		\end{figure}
+		\item Now your dataframe is defined in the variable \texttt{df}. You can use that name to refer to it or print it, as in:
+		\begin{figure}
+			\includegraphics[width=0.8\linewidth]{img/df.png}
+		\end{figure}
+	\end{enumerate}
+
+\end{frame}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Slide
+\begin{frame}{Pandas}
+
+	\textbf{Creating a dataframe}
+
+	Another way to create a dataframe from zero is to define an empty dataframe and then create its columns individually.
+	\begin{figure}
+		\includegraphics[width=0.8\linewidth]{img/df_definition2.png}
+	\end{figure}
+
+\end{frame}
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Section title slide
+\sectionpic{Reading and exploring data}{img/section_slide}
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Slide
+\begin{frame}{Reading and exploring data}
+
+	\textbf{Reading data}
+
+	\begin{itemize}
+		\item In our work, we don't usually need to define a dataframe from scratch
+		\item More often, we load pre-existing data files
+		\item To load a \texttt{.csv} file into a dataframe, we use the command \texttt{read\_csv()}
```

Comment:
Can we provide examples for reading dtas and savs?

Reply:
added in 25e438c817abc3c080e90ee7a78a5f0aa4ad0ade